plugins/catalog-node/src/alpha.ts

```diff
@@ -17,3 +17,5 @@
 export { catalogServiceRef } from './catalogService';
 export type { CatalogProcessingExtensionPoint } from './extensions';
 export { catalogProcessingExtensionPoint } from './extensions';
+export * from './processing';
+export * from './api';
```

Comment:
These aren't alpha though. They should all go in the regular types instead.

If you were hitting problems with importing from alpha code to non-alpha code in the same library, what you need to do is to do it via the package name instead of via local paths, as if you were an external plugin. I don't expect the alpha types to change because of this move of things to the node package.